False negative on EveryLB support test

```
CMake Error at cmake/FindCharm.cmake:63 (message):
  Could not find charm module EveryLB.Make sure you have built the LIBS
  target when building charm++
Call Stack (most recent call first):
  cmake/SetupCharm.cmake:4 (find_package)
  CMakeLists.txt:70 (include)
```
But if I remove the CMake check the code compiles and runs.

Probably the issue is that it only checks for a static library.

--- a/cmake/FindCharm.cmake
+++ b/cmake/FindCharm.cmake
@@ -53,18 +53,19 @@ find_library(CHARM_LIBCK
     PATH_SUFFIXES lib
     HINTS ${CHARM_ROOT} ENV CHARM_ROOT
 )
-find_library(CHARM_EVERYLB
-    NAMES libmoduleEveryLB.a
-    PATH_SUFFIXES lib64 lib
-    HINTS ${CHARM_ROOT} ENV CHARM_ROOT
-)
-
-if("${CHARM_EVERYLB}" STREQUAL "CHARM_EVERYLB-NOTFOUND")
-  message(SEND_ERROR "Could not find charm module EveryLB."
-    "Make sure you have built the LIBS target when building charm++")
-endif()
 
 get_filename_component(CHARM_LIBRARIES ${CHARM_LIBCK} DIRECTORY)
+
+find_library(CHARM_EVERYLB
+  NAMES moduleEveryLB
+  HINTS ${CHARM_LIBRARIES}
+  NO_DEFAULT_PATH
+  )
+
+if("${CHARM_EVERYLB}" STREQUAL "CHARM_EVERYLB-NOTFOUND")
+  message(SEND_ERROR "Could not find charm module EveryLB. "
+    "Make sure you have built the LIBS target when building charm++")
+endif()
 
 find_program(CHARM_COMPILER
   NAMES charmc

